app/scripts/models/account.js

```diff
@@ -249,10 +242,44 @@ define(function (require, exports, module) {
       return this._fxaClient.recoveryEmailStatus(sessionToken);
     },
 
-    isVerified: function () {
```

Comment:
This function didn't really have a reason to exist when `sessionStatus` is so close to it in functionality.